DatadogFlags/Tests/Feature/FlagsClientMocks.swift

```diff
@@ -0,0 +1,121 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+import DatadogInternal
+import TestUtilities
+
+@testable import DatadogFlags
+
+extension FlagsEvaluationContext: AnyMockable, RandomMockable {
+    public static func mockAny() -> FlagsEvaluationContext {
+        .init(targetingKey: .mockAny(), attributes: .mockAny())
+    }
+
+    public static func mockRandom() -> FlagsEvaluationContext {
+        .init(targetingKey: .mockRandom(), attributes: .mockRandom())
+    }
+}
+
+extension FlagAssignment: AnyMockable, RandomMockable {
+    public static func mockAny() -> FlagAssignment {
+        .mockAnyBoolean()
+    }
+
+    public static func mockRandom() -> FlagAssignment {
+        .init(
+            allocationKey: .mockRandom(),
+            variationKey: .mockRandom(),
+            variation: .mockRandom(),
+            doLog: .mockRandom()
+        )
+    }
+
+    static func mockAnyBoolean(doLog: Bool = true) -> FlagAssignment {
+        .init(
+            allocationKey: .mockAny(),
+            variationKey: .mockAny(),
+            variation: .mockAnyBoolean(),
+            doLog: doLog
+        )
+    }
+
+    static func mockAnyString(doLog: Bool = true) -> FlagAssignment {
```

Comment:
Some of these (`mockAnyString`, `mockAnyInteger`, `mockAnyDouble`, `mockAnyObject`) appear to be unused, but I guess they're here for completeness